.ci/pipeline_definitions

```diff
@@ -1,24 +1,12 @@
 gardener-website-generator:
-  template: 'default'
   base_definition:
-    traits:
-      version: ~
-      scheduling:
-        suppress_parallel_execution: true
-    repo:
```

Comment:
pipeline repository is added as "main source" by default anyways, therefore you can omit it